Content.Server/Decals/DecalSystem.cs

```diff
@@ -307,150 +312,63 @@ public HashSet<(uint Index, Decal Decal)> GetDecalsInRange(EntityUid gridId, Vec
 
                 if (validDelegate == null || validDelegate(decal))
                 {
-                    uids.Add((uid, decal));
+                    decalIds.Add((uid, decal));
                 }
             }
 
-            return uids;
+            return decalIds;
         }
 
-        public bool SetDecalPosition(EntityUid gridId, uint uid, EntityCoordinates coordinates)
+        /// <summary>
+        ///     Changes a decals position. Not this will actually result in a new decal being created, possibly on a new grid or chunk.
```

Comment:
```suggestion
        ///     Changes a decals position. Note this will actually result in a new decal being created, possibly on a new grid or chunk.
```